src/main/resources/export/java/RobotContainer-constructors.java

```diff
@@ -17,4 +17,17 @@
 #end
 #end
 
-
+${Collections.reverse($components)}
+        // SmartDashboard Buttons
+#foreach( $component in $components )
+#if ($component.getBase().getType() == "Command"
+     && $component.getProperty("Button on SmartDashboard").getValue())
+#if( $component.getProperty("Parameter presets").getValue().isEmpty() )
+        SmartDashboard.putData("$component.getName()", #command_instantiation_SmartDashoard_BTN_No_Par($component.getName(), $component));
```

Comment:
This can be simplifed to use existing macros
```suggestion
        SmartDashboard.putData("$component.getName()", #new_command_instantiation_nt($component, $component.getProperty("Parameter presets").getValue()));
```